meshtastic/mt_config.py

```
@@ -8,32 +7,127 @@
+
+def __getattr__(name: str) -> Any:
+    """Provide backwards-compatible access to renamed module attributes."""
+    if name in _COMPAT_ALIASES:
+        new_name = _COMPAT_ALIASES[name]
+        warnings.warn(
+            f"mt_config.{name} is deprecated, use mt_config.{new_name} instead",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return globals()[new_name]
+    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)